tests/python/unittest/test_gluon_data_vision.py

```diff
@@ -83,6 +83,25 @@ def test_transformer():
     transform(mx.nd.ones((245, 480, 3), dtype='uint8')).wait_to_read()
 
 
+@with_seed()
+def test_batchify():
+    transformer = transforms.Batchify()
+    im1 = nd.random.uniform(0, 1, (3, 28, 28))
+    im2 = nd.random.uniform(0, 1, (3, 28, 28))
+    im3 = nd.random.uniform(0, 1, (3, 28, 28))
```

Comment:
Could you add a test with inputs of different sizes too? That may serve as an example for users as well

Reply:
can the batch sizes be randomized?